daemon/main.c

```diff
@@ -742,20 +742,20 @@ void send_statistics( const char *action, const char *action_result, const char
         if (likely(access(optout_file, R_OK) != 0)) {
             as_script = mallocz(sizeof(char) * (strlen(netdata_configured_primary_plugins_dir) + strlen("anonymous-statistics.sh") + 2));
             sprintf(as_script, "%s/%s", netdata_configured_primary_plugins_dir, "anonymous-statistics.sh");
-			if (unlikely(access(as_script, R_OK) != 0)) {
-				netdata_anonymous_statistics_enabled=0;
-				info("Anonymous statistics script %s not found.",as_script);
-				freez(as_script);
-			} else {
-				netdata_anonymous_statistics_enabled=1;
-			}
-		} else {
+                        if (unlikely(access(as_script, R_OK) != 0)) {
```

Comment:
why additional indentation for that if/else?

Reply:
I think I know the problem, because I had something like this, the Clion uses spaces to do tabulation instead tab, so this can happen because the writer were using tabs or the computer uses 8 spaces instead 4 to tabulation. I had problem with the second.

Reply:
its not additional indent. There were tabs and switched them to spaces to match properly

Reply:
Now i saw what ilya meant, i just pushed the fix.
In the first place the code had extra indent already.
I thought he meant that my change introduced extra indent!

fixed, thanks ilya!